adaptors/dummy/deploy/e2e/usability/usability_test.go

```diff
@@ -0,0 +1,580 @@
+package usability_test
+
+import (
+	"path/filepath"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	apiv1alpha1 "github.com/rancher/octopus/adaptors/dummy/api/v1alpha1"
+	edgev1alpha1 "github.com/rancher/octopus/api/v1alpha1"
+	"github.com/rancher/octopus/pkg/util/object"
+	"github.com/rancher/octopus/test/util/content"
+	"github.com/rancher/octopus/test/util/exec"
+	"github.com/rancher/octopus/test/util/node"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	apierrs "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
Please use `goimports` to format the import block.

Reply:
updated